wiki/versions/1.19/data/recipes/groovy/src/main/groovy/org/quiltmc/wiki/recipes/MyRecipe.groovy

```diff
@@ -0,0 +1,126 @@
+package org.quiltmc.wiki.recipes
+
+import com.google.gson.JsonObject
+import groovy.transform.CompileStatic
+import net.minecraft.inventory.Inventory
+import net.minecraft.item.ItemStack
+import net.minecraft.network.PacketByteBuf
+import net.minecraft.recipe.Ingredient
+import net.minecraft.recipe.Recipe
+import net.minecraft.recipe.RecipeSerializer
+import net.minecraft.recipe.RecipeType
+import net.minecraft.util.Identifier
+import net.minecraft.util.JsonHelper
+import net.minecraft.util.registry.Registry
+import net.minecraft.world.World
+import org.quiltmc.qsl.recipe.api.serializer.QuiltRecipeSerializer
+
+class MyRecipe implements Recipe<Inventory> {
+    // @start Starting
+    private final Identifier id
```

Comment:
This seems like a prime place to leverage final public properties (for the output) and `@TupleConstructor`